packages/api/src/schema/schema.yaml

```diff
@@ -1363,6 +1363,12 @@ components:
             errorMessage:
               type: string
               description: Error message if the task failed
+            attempts:
+              type: number
+              description: Number of attemps done on the task
```

Comment:
I think we should either name this `attempt` and always have it there and have them numbered starting from 1, to be read as an ordinal value ("first attempt", "second attempt", etc)

Or we can rename this to `retries` instead to have it mean "number of retries that were needed" or something like that, which then makes sense to start at `0` (and then not having it present on the first attempt). Kinda preferring this one, WDYT?